packages/block-library/src/image/edit.native.js

```diff
@@ -496,6 +496,8 @@ export class ImageEdit extends Component {
 			imageDefaultSize,
 		] );
 
+		const canImageBeFeatured = typeof attributes.id !== 'undefined';
```

Comment:
Let's add a comment to elaborate on what's going on here. I mean, `canImageBeFeatured` has a descriptive name by itself but, why is it that if the image block doesn't have an id it can't become Featured?

Reply:
Makes sense! Add a comment now. :)